export const ErrorMessages = {
    Internal: {
        UNEXPECTED_ERROR: "Ocorreu um erro inesperado. Por favor, tente novamente mais tarde.",
        INTERNAL_ERROR: "Ocorreu um erro interno no servidor. Por favor, tente novamente mais tarde.",
    },
    Validation: {
        MESSAGE_NOT_REGISTERED: "Mensagem n√£o cadastrada.",
        VALUE_NOT_FOUND: "Nenhum valor foi encontrado.",
    },
    failure: {
        transactionCreation: 'Nenhum transaction foi criada',
    }
} as const;